### ISSUE TITLE
Query with two patterns fails

### ISSUE THREAD

**Issue by pheyvaer** [comunica/comunica-actor-init-sparql-hdt#94]
The following query doesn't return results
```
SELECT * WHERE {?a a ?c; ?p ?o.}
```
while the input is 
```
<http://example.com> a <http://example.com/Test>; 
  <http://example.com/name> "Test".
```
which is converted to HDT.
The query is executed using the following command

```
comunica-sparql-hdt hdtFile@test.hdt "SELECT * WHERE {?a a ?c; ?p ?o.}"
```

The following query gives all expected results though
```
SELECT * WHERE {?a ?b ?c}
```
Details about my installation:

| software            | version
| ------------------- | -------
| Comunica Init Actor | 1.17.0 
| node                | v12.18.4
| npm                 | 6.14.8
| yarn                | _Yarn is unavailable_
| Operating System    | darwin (Darwin 19.6.0)

**Comment by rubensworks** [comment]
Some big changes were made to the internal asynciterators, which may also require some small changes in this package. I'll look into it soon.

In the meantime, I would suggest downgrading to Comunica 1.16.x when using this HDT package.

**Comment by pheyvaer** [comment]
I cloned the repo and updated the package.json to use version 1.16.x. However, how do I build this? I did
```
yarn run prepare
yarn run build
```

But when I execute the same command to execute the query I get 

```
this.sources.map is not a function
```

No further output is provided.

**Comment by rubensworks** [comment]
Fixed in release 1.7.
